cluster-scope/base/groups/groups.yaml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Given how the groups file is organized aren't we basically saying that if you import this groups folder, then you _have_ have to  implement all these groups regardless of cluster?

Reply:
Yes we are saying that... Kinda. Would you like me to change that? It's a bit of inconsistency, right? Yeah, let's change it. :thinking:

Reply:
I think removing the possibility to include "all" of a certain kind to a specific environment would also make it less error prone. Good call. Changed.